docs/security-considerations.rst

```diff
@@ -4,31 +4,25 @@
 Security Considerations
 #######################
 
-While it is usually quite easy to build software that works as expected,
-it is much harder to check that nobody can use it in a way that was **not** anticipated.
-
-In Solidity, this is even more important because you can use smart contracts
-to handle tokens or, possibly, even more valuable things. Furthermore, every
-execution of a smart contract happens in public and, in addition to that,
-the source code is often available.
-
-Of course you always have to consider how much is at stake:
-You can compare a smart contract with a web service that is open to the
-public (and thus, also to malicious actors) and perhaps even open source.
-If you only store your grocery list on that web service, you might not have
-to take too much care, but if you manage your bank account using that web service,
-you should be more careful.
-
-This section will list some pitfalls and general security recommendations but
-can, of course, never be complete.  Also, keep in mind that even if your smart
-contract code is bug-free, the compiler or the platform itself might have a
-bug. A list of some publicly known security-relevant bugs of the compiler can
-be found in the :ref:`list of known bugs<known_bugs>`, which is also
-machine-readable. Note that there is a bug bounty program that covers the code
-generator of the Solidity compiler.
-
-As always, with open source documentation, please help us extend this section
-(especially, some examples would not hurt)!
+While it is usually quite easy to build software that works as expected, it is much harder to check that nobody can use it in a way that was **not** anticipated.
```

Comment:
I do not want to enter in a long discussion about the topic. However, I'm curious about the reason of these changes. Why not keep the line length less than 80 characters? Or even 130 max? 161 seems to much in my opinion.

Reply:
Readability for contributors. In docs, you either break on commas or in the end of sentences, not mid-sentence.

Reply:
hum...IMHO a markup language like `reStructuredText` should be treated as code when not in its rendered form, since whoever is editing it would benefit of having an entire readable text block that fits the eyes sight, instead of long text lines that may be difficult to follow. And, shorter lines also helps to clearly identify the changes per line when reviewing a PR, for example.
This is, of course, based on the premise that the optimal line length for readability of a text is in between 50 to 80 characters.

Reply:
Text wrap is a reality in most code editors, making your eyes jump lines while reading text, unfortunately, is not. 😅

This is very common in contributor usability on documentation for that purpose.

Reply:
Also take a look at the original line 197. 😉

```
before they interact with your contract. Note that, since `Tangerine Whistle <https://eips.ethereum.org/EIPS/eip-608>`_ hardfork, the `63/64 rule <https://eips.ethereum.org/EIPS/eip-150>`_ makes call stack depth attack impractical. Also note that the call stack and the expression stack are unrelated, even though both have a size limit of 1024 stack slots.
```